docs: remix section in tools (#1003)
oasisprotocol/docs: docs/dapp/emerald/writing-dapps-on-emerald.mdx

This tutorial will show you how to set up dApp development environment for
Emerald to be able to write and deploy dApps on Oasis Emerald.
Oasis Emerald exposes an **EVM-compatible** interface so writing dApps isn't much
different compared to the original Ethereum Network!

We will walk you through the Hardhat configuration.

> Those who prefer a simpler web-only interface can also the Remix IDE, check our
> general [Remix guide] for that. Just remember to use the Emerald [networks]
> when selecting *Inject Web3* environment and connecting with your MetaMask.

Reviewer: ```suggestion
Those who prefer a simpler web-only interface can also use the Remix IDE. Check out our
general [Remix guide]. Just remember to use the Emerald [networks]
when selecting *Inject Web3* environment and connecting to MetaMask.
```
